gencert-private.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/sh
+
+# generate configuration file
+
+openssl_conf=$(mktemp)
+
+cat > "${openssl_conf}" <<EOF
+oid_section=spid_oids
+
+[ req ]
+default_bits=2048
+default_md=sha256
```

Comment:
2048 bits and sha256 are the minimum requirements. I would suggest to use something a bit more secure like 3072 bits keys and sha384 digest. Alternatively, these values could be managed with envvars (and default values) like in the following
```
[req]
default_bits=${KEY_LENGTH:=2048}
default_md=${MD_ALG:=sha256}
```